requirements.opt.txt

```diff
@@ -1,12 +1,9 @@
 cffi
-torchvision
 joblib
-librosa
 numba==0.43.0
 llvmlite==0.32.1
-Pillow
-git+git://github.com/pytorch/audio.git@d92de5b97fc6204db4b1e3ed20c03ac06f5d53f0
 pyrouge
-opencv-python
 git+git://github.com/NVIDIA/apex.git@700d6825e205732c1d6be511306ca4e595297070
 pretrainedmodels
```

Comment:
This one can be removed, as it was only used for tools/vid_feature_extractor.py, which is removed for now.

Reply:
THX for reminding.